examples/annotation_import/conversational.ipynb

```diff
@@ -33,16 +33,17 @@
         "# Conversational Text Annotation Import\n",
```

Comment:
<p>Can you please uncomment this code, now that we do not use relationships, we can use Label Import</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>